src/traversals/bipartiteness.jl

```diff
@@ -0,0 +1,48 @@
+"""
```

Comment:
Bipartition sounds like it takes a graph and identifies the parts. I guess that is what it does. So that works.